Assets/Scripts/Models/WorldGenerator.cs

```diff
@@ -80,12 +80,12 @@ public static void Generate(World world, int seed)
 
         for (int z = 0; z < depth; z++)
         {
-            float zScale = Mathf.Lerp(1f, .5f, Mathf.Abs((depth / 2f) - z) / depth);
```

Comment:
I've got this (something bad on me setup ?)
SA1305 : CSharp.Naming : The variable name 'zScale' begins with a prefix that looks like Hungarian notation. Remove the prefix or add it to the list of allowed prefixes.